pimd/pim_cmd_common.c

```diff
@@ -3232,9 +3266,9 @@ void pim_show_neighbors(struct pim_instance *pim, struct vty *vty,
 						       neigh_src_str, json_row);
 
 			} else {
-				vty_out(vty, "%-16s  %15s  %8s  %8s  %6d\n",
-					ifp->name, neigh_src_str, uptime,
-					expire, neigh->dr_priority);
+				ttable_add_row(tt, "%s|%s|%s|%s|%d", ifp->name,
+					       neigh_src_str, uptime, expire,
```

Comment:
Can we reuse `%pPAs` for `&neigh->source_addr`?

Reply:
Fixed